code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -246,6 +246,15 @@
 	description = "A salt made of sodium chloride. Commonly used to season food."
 	reagent_state = SOLID
 	color = "#FFFFFF" // rgb: 255,255,255
+	overdose_threshold = 30
+
+/datum/reagent/consumable/sodiumchloride/overdose_process(mob/living/M)
+	var/high_message = pick("You feel really upset.", "You feel really cranky.", "You feel like crying.", "You feel like shouting about how unfair your life is.")
+	if(prob(10))
```

Comment:
Please put the message inside this if. No need to do a pick operation if you're gonna ignore it 90% of the time